tools/deps/prebuilt/include/dispatch/queue_internal.h

```diff
@@ -81,23 +99,25 @@ extern struct dispatch_queue_s _dispatch_root_queues[];
 void _dispatch_queue_init(dispatch_queue_t dq);
 void _dispatch_queue_drain(dispatch_queue_t dq);
 void _dispatch_queue_dispose(dispatch_queue_t dq);
-void _dispatch_queue_push_list_slow(dispatch_queue_t dq, struct dispatch_object_s* obj);
+void _dispatch_queue_push_list_slow(dispatch_queue_t dq, struct dispatch_object_s *obj);
 void _dispatch_queue_serial_drain_till_empty(dispatch_queue_t dq);
 void _dispatch_force_cache_cleanup(void);
 
 DISPATCH_INLINE
-static void _dispatch_queue_push_list(dispatch_queue_t dq, dispatch_object_t _head, dispatch_object_t _tail) {
-    struct dispatch_object_s *prev, *head = _head._do, *tail = _tail._do;
-
-    tail->do_next = NULL;
-    prev = fastpath(dispatch_atomic_xchg_pointer(&dq->dq_items_tail, tail));
-    if (prev) {
-        // if we crash here with a value less than 0x1000, then we are at a known bug in client code
-        // for example, see _dispatch_queue_dispose or _dispatch_atfork_child
-        prev->do_next = head;
-    } else {
-        _dispatch_queue_push_list_slow(dq, head);
-    }
+static void
+_dispatch_queue_push_list(dispatch_queue_t dq, dispatch_object_t _head, dispatch_object_t _tail)
```

Comment:
please revert these formatting changes.

Reply:
This is actually _reverting_ formatting changes we made to prebuilt headers years ago :)